### PR TITLE
Shutter/curtain device

### PR DESCRIPTION
**Pull request by atrovato**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ ] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.
- ~~[ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)~~
- ~~[ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).~~
- [x] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Add curtain/shutter feature.
Not used by services for now.

### CODE REVIEW

**Review by Pierre-Gilles — COMMENTED** [review]
Je ne trouve pas la feature dans l'intégration MQTT 

Screenshot 2022-07-28 at 15 30 13

**Comment by atrovato** [comment]
I fixed it for cover devices (shutter and curtain), BUT we have same problème if we looking for "capteur", wich is a category word, not a feature.

The problem is more on the select element and its filter feature. It filters only on selectable elements, no on matching parent group.

**Comment by Pierre-Gilles** [comment]
@atrovato Nice! 

Could you make the UX a bit better when the button are clicked? There are no feedbacks right now, and it's hard to know if it's working or if it's bugged! 

See video: 

https://user-images.githubusercontent.com/7365207/182812953-debbdde8-f40f-43d0-9e4b-623f00ddb512.mov

File: front/src/components/boxs/device-in-room/device-features/CoverDeviceFeature.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import get from 'get-value';
+import { Text } from 'preact-i18n';
+
+import { getDeviceName } from './utils';
+import { DeviceFeatureCategoriesIcon } from '../../../../utils/consts';
+import { COVER_STATE } from '../../../../../../server/utils/constants';
+
+const UpDownDeviceFeature = ({ children, ...props }) => {
```
Comment (VonOx) [inline front/src/components/boxs/device-in-room/device-features/CoverDeviceFeature.jsx]: Must be CoverDeviceFeature  ?

**Comment by atrovato** [comment]
I add the 'active' class to keep last selected state (and last received state) as selected button.

**Review by Pierre-Gilles — APPROVED** [review]
Looks fine to me with the feedback, thanks for this PR ! 🙏

**Pull request merged**